src/config/rcxml.c

```diff
@@ -906,6 +912,19 @@ entry(xmlNode *node, char *nodename, char *content)
 		rc.window_edge_strength = atoi(content);
 	} else if (!strcasecmp(nodename, "range.snapping")) {
 		rc.snap_edge_range = atoi(content);
+	} else if (!strcasecmp(nodename, "edgeType.previewTimeout.snapping")) {
+		if (!strcasecmp(content, "interior")) {
+			edge_type = EDGE_TYPE_INTERIOR;
+		} else if (!strcasecmp(content, "exterior")) {
+			edge_type = EDGE_TYPE_EXTERIOR;
+		}
```

Comment:
Can we come up with a more intuitive config name for non-devs (regarding `interior` and `exterior`) ?
If we ship this in the next release we really don't want to ever change it to something else so now would be the time.

Just to reiterate: this allows changing the timeout based on the fact that there is another adjacent output in the direction of the edge.

Reply:
My random ideas are:
- `edgePosition="inner|outer"`
- `edgeType="withAdjacentOutput|withoutAdjacentOutput"`

Reply:
`inner | outer` seems easier to understand to me. Thoughts @johanmalm ?

Reply:
- `shared` and `layoutPerimeter`
- `betweenOutputs` and `perimeterOfOutputLayout'

Reply:
I like `edgeType="inner|outer"`, although "shared" and one of "exterior" or "boundary" would be acceptable alternatives. No matter what, we'll have to explain the meaning in the documentation, and I think there's enough mnemonic value in any of these labels to be understood.